api/krusty/fnplugin_test.go

```diff
@@ -32,23 +68,117 @@ stringData:
     bootcmd:
     - mkdir /mnt/vda
 `)
+	th.WriteF(filepath.Join(tmpDir.String(), "exec.sh"), execFile)
```

Comment:
`"generateDeployment.sh"`

sorry to pick nits, just going for more readability here.